tests/helpers/start-server.js

```diff
@@ -36,7 +37,10 @@ module.exports = function runServer(callback, options) {
   args.push(commandOptions);
 
   return new RSVP.Promise(function(resolve, reject) {
-    runCommand.apply(null, args)
+    runCommand.call(null, emberCommand, 'build') // build 'dist'
```

Comment:
So this reflects a real-world problem: you need to now run `ember build` (probably `ember build -prod`) before you run `ember fastboot --serve-assets`, because we need to serve the browser builds from `dist`. (See the commit message for details)

Unfortunately, it is not easy to refactor the set up to produce both builds in a single command (it relies on a global ENV variable, among other things).

So I just added a new `assets-path` option for now, but we should revisit this in the future.